pennylane/ops/qubit/__init__.py

```diff
@@ -70,6 +70,7 @@
     "RY",
     "RZ",
     "PhaseShift",
+    "PCPhase",
```

Comment:
Please also make the gate appear in [Quantum operators](https://docs.pennylane.ai/en/stable/introduction/operations.html).

Reply:
Nice catch, added 👍🏼